docs/metrics/customizing-the-sdk/README.md

```diff
@@ -335,6 +335,61 @@ by using Views.
 
 See [Program.cs](./Program.cs) for a complete example.
 
+### Explicit Bucket Histogram Aggregation with .NET 9 Advice API
```

Comment:
It’s misleading to call it .net 9 advice api as this api can be used in older .net runtimes

Reply:
This is unresolved.